pkg/security/ptracer/syscalls_arm64.go

```diff
@@ -21,15 +21,19 @@ const (
 	CloneNr    = 220 // CloneNr defines the syscall ID for arm64
 	ExitNr     = 93  // ExitNr defines the syscall ID for arm64
 	FcntlNr    = 25  // FcntlNr defines the syscall ID for arm64
-	DupNr      = 23  // DupNr defines the syscall ID for amd64
-	Dup3Nr     = 24  // Dup3Nr defines the syscall ID for amd64
-	ChdirNr    = 49  // ChdirNr defines the syscall ID for amd64
-	FchdirNr   = 50  // FchdirNr defines the syscall ID for amd64
+	DupNr      = 23  // DupNr defines the syscall ID for arm64
+	Dup3Nr     = 24  // Dup3Nr defines the syscall ID for arm64
+	ChdirNr    = 49  // ChdirNr defines the syscall ID for arm64
+	FchdirNr   = 50  // FchdirNr defines the syscall ID for arm64
+	SetuidNr   = 146 // SetuidNr defines the syscall ID for arm64
+	SetgidNr   = 144 // SetgidNr defines the syscall ID for arm64
+	SetreuidNr = 145 // SetreuidNr defines the syscall ID for arm64
+	SetregidNr = 143 // SetregidNr defines the syscall ID for arm64
 
-	OpenNr  = 9990 // OpenNr not available on arm64
-	ForkNr  = 9991 // ForkNr not available on arm64
-	VforkNr = 9992 // VforkNr not available on arm64
-	Dup2Nr  = 9993 // Dup2Nr not available on arm64
+	OpenNr  = -1 // OpenNr not available on arm64
+	ForkNr  = -1 // ForkNr not available on arm64
+	VforkNr = -1 // VforkNr not available on arm64
+	Dup2Nr  = -1 // Dup2Nr not available on arm64
```

Comment:
As these values are used in a switch case it wont compile (that's why I've given them different values initialy)